templates/bootstrap.yaml

```diff
@@ -29,25 +29,6 @@ Parameters:
     ConstraintDescription: 'Must be an acceptable email address syntax(i.e. joe.smith@sagebase.org)'
     Default: 'it@sagebase.org'
 Resources:
-  # !! IMPORTANT !! - AWS API will refuse to remove users that have attached resources.
-  # Therefore you must do the following before deleting them from this file:
-  # 1. Detach or remove the following user resources: login profile, attached
-  #    MFA device, access-keys, ssh-keys, and policies.
-  # 2. Detach the user from all groups.
-  AWSIAMTravisUserAccessKey:
```

Comment:
Don't we use this key in this repo for pushing the templates to S3?

Reply:
i believe we actually use the role this template from https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/sceptre/admincentral/templates/service-accounts.yaml